### DISCUSSION TITLE
manual correction

### DISCUSSION THREAD

**Discussion by Micha-65** [jomjol/AI-on-the-edge-device#1614] [General]
### The Feature

Is there somewhere to tell the system that the detected value is wrong and enter a correction?
a workout?
If yes, where?
I use version 13.0.7 with a gas meter.
Thanks Micha

**Comment by caco3** [comment]
Use the menu `Settings > Set Previous Value`

**Reply by Micha-65** [reply]
But, does the system learn, for example, by the fact that the eight at this point is a six?

**Reply by caco3** [reply]
Ah, you mean that it recognizes the number wrong?

- You could try a different model, see https://github.com/jomjol/AI-on-the-edge-device/wiki/Choosing-the-Model
- If some number still are wrong, check if your ROIs are setup ok and the digits are already contained in the database, see https://github.com/jomjol/AI-on-the-edge-device/wiki/ROI-Configuration

**Reply by Micha-65** [reply]
Yes, exactly, I see the correct number on the display, but he gets it wrong every time.
Then you should be able to tell him what is correct.
I have now changed something on the model, we'll see.
I'm also looking for something against the reflections, that would also help a lot.
hp my english is not good, i will take a translate.
Excuse me.
German would be better ;-)

**Comment by jomjol** [comment]
The model is a trained neural network. It cannot be trained on the ESP32. Even on modern computers it takes quite some time (30-90 Minutens). The way to go is: collect wrong detected images, label them and support them as training data.

You can do also your own training: https://github.com/jomjol/AI-on-the-edge-device/wiki/Learn-models-with-your-own-images
